src/app/Instance/InstanceDetails/InstanceDetails.tsx

```diff
@@ -91,6 +95,22 @@ export const InstanceDetails = ({
               {instance.published_at && formatDate(instance.published_at)}
             </DescriptionListDescription>
           </DescriptionListGroup>
+          <DescriptionListGroup>
+            <DescriptionListTerm>
+              {t("instance.cloudProvider")}
+            </DescriptionListTerm>
+            <DescriptionListDescription data-ouia-component-id="instance-details-cloud-provider">
+              {cloudProvider?.label ?? "Not available"}
```

Comment:
Like suggested by Valentino, we should leave no untranslated strings inside templates.

Could you please add a translation for `Not available`?

You'll have to add a new key/value pair inside the dictionary located at `locales/en/openbridge.json`, inside the `common` list I would say.

Reply:
The changes have been made.